src/Common/ThreadStatus.h

```diff
@@ -1,12 +1,12 @@
 #pragma once
 
-#include <Core/SettingsEnums.h>
-#include <Interpreters/Context_fwd.h>
+#include <Core/LogsLevel.h>
```

Comment:
Make ThreadStatus.h (included in many places) not depend on SettingsEnums.h (pretty heavy)